src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/Utilities/BaseTest.cs

```diff
@@ -401,14 +404,12 @@ protected virtual void CleanupTest ()
 		{
 			TestContext.Out.WriteLine ($"[TESTLOG] Test {TestName} Complete");
 			TestContext.Out.Flush ();
-			if (System.Diagnostics.Debugger.IsAttached || TestContext.CurrentContext.Test.Properties ["Output"] == null)
+			var outputDir = TestOutputDirectories [TestContext.CurrentContext.Test.ID];
```

Comment:
So this will probably need to use TryGetValue now, somehow NUnit had a custom dictionary before that returned null when the key was not found?